### DISCUSSION TITLE
[Suggestion] Could /seen display months as well?

### DISCUSSION THREAD

**Discussion by mrfloris** [Zrips/CMI#3091] [Feature Requests]
Currently it seems if someone's been away over 365 days, it becomes 1 year. It doesn't however say that 37 days ago was 1 month and 1 week ago.

It has year / days / hours
It doesn't seem to have years / months / weeks / days / hours.

**Comment by Zrips** [comment]
In general its to shorten output line. Having something like: **1 year 5 months 6 days 3 hours 25 minutes 55 seconds** can be quite long line, we could even get rid of years and end up with basic: **521 days 3 hours 25 minutes 55 seconds** and its using same generic time formating which is used across entire plugin for simplicity sake. If you have a better idea how to show bigger dates in relatively short format, i will gladly change it.

**Comment by mrfloris** [comment]
I think seconds and minutes is kinda meh data when it comes to years and months. 
<img width="1001" alt="Screen-Shot 2022-09-15 at 17 27 28" src="https://user-images.githubusercontent.com/28841349/190444408-f21cdc36-17ef-41b9-91d3-af277e3a282e.png">
I think having 
1 year, 4 months, and 1 week ago, is short enough and gives a good impression for ppl who haven't been around in ages.
plus, hovering over it json tooltip could give it in full details. from years, months, weeks, days, minutes, second. and even have other details as well, like first login date, and POS: xyz, if they have the perm, and something like "sessions: 31 times logged in"
